README.md

```
@@ -298,6 +355,12 @@ Execute the following inside the bosh-acceptance-tests directory:
 bundle exec rspec spec --tag ~vip_networking --tag ~dynamic_networking --tag ~root_partition --tag ~raw_ephemeral_storage
 ```
 
+Here is the same for Proxmox VE on a lab with one IPv4 network. PVE has no floating IP concept and no raw instance storage, and its vm_types size the root disk explicitly, so those tags are skipped along with the ones the single network cannot cover:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)